.circleci/config.yml

```diff
@@ -369,7 +392,7 @@ commands:
             fi
           done
           "$xur_path" "${args[@]}"
-        no_output_timeout: 65s
+        no_output_timeout: 120s
```

Comment:
Does this significantly help unresponsive tests to pass? I know the value is / was based on experience, but wonder if increasing the value actually help.

Reply:
Not significantly, the errors that I wanted to remove wasn't so much affect by those changes.